overview/http_server.md

```diff
@@ -3,15 +3,16 @@
 A slightly more interesting example is an HTTP Server:
 
 ```crystal
+
 require "http/server"
 
-server = HTTP::Server.new(8080) do |context|
-  context.response.content_type = "text/plain"
-  context.response.print "Hello world! The time is #{Time.now}"
+server = HTTP::Server.new do |context|
+  context.response.headers["Content-Type"] = "text/plain"
+  context.response.print("Hello world! The time is #{Time.now}")
 end
 
-puts "Listening on http://127.0.0.1:8080"
-server.listen
+puts "Listening on http://0.0.0.0:8080"
```

Comment:
If it is supposed to print the listening address, I'd rather do this:
```cr
address = server.bind_tcp "0.0.0.0", 8080
puts "Listening on #{address}"
server.listen
```
This is more DRY and in some cases the actual address might be different from the arguments delivered to `listen`.

Reply:
That's preferrable.